tests/unit/plugins/modules/test_modprobe.py

```diff
@@ -174,3 +180,427 @@ def test_unload_module_failure(self):
         module.fail_json.assert_called_once_with(
             msg='', rc=1, stdout='', stderr='', **dummy_result
         )
+
+
+class TestModuleIsLoadedPersistently(ModuleTestCase):
+    def setUp(self):
+        if (sys.version_info[0] == 3 and sys.version_info[1] < 7) or (sys.version_info[0] == 2 and sys.version_info[1] < 7):
+            self.skipTest('open_mock doesnt support readline in earlier python versions')
+
+        super(TestModuleIsLoadedPersistently, self).setUp()
+
+        self.mock_get_bin_path = patch('ansible.module_utils.basic.AnsibleModule.get_bin_path')
+
+        self.get_bin_path = self.mock_get_bin_path.start()
+
+    def tearDown(self):
+        """Teardown."""
+        super(TestModuleIsLoadedPersistently, self).tearDown()
+
+        self.mock_get_bin_path.stop()
+
+    def test_module_is_loaded(self):
+
+        set_module_args(dict(
+            name='dummy',
+            state='present',
+            persistent='present'
+        ))
+
+        module = AnsibleModule(
+            argument_spec=dict(
+                name=dict(type='str', required=True),
+                state=dict(type='str', default='present', choices=['absent', 'present']),
+                params=dict(type='str', default=''),
+                persistent=dict(type='str', default='disabled', choices=['disabled', 'present', 'absent']),
+            ),
+            supports_check_mode=True,
+        )
```

Comment:
Moving the `AnsibleModule` creation code to a function (best somewhere in `plugins/modules/modprobe.py` itself) would make this less repetetive.